README.md

```diff
@@ -1,256 +1,170 @@
 
-  this.$('button').click();
-});
+```sh
+$ ember install ember-qunit
 ```
 
-Component integration tests are the default mode for `moduleForComponent`. You can still explicitly activate them by passing `integration: true`.
-
-Integration tests have the advantage of testing your component as Ember would actually use them.  It's helpful to think of this mode as simply testing the inputs and outputs of the component.  These tests allow you interact with both the bound values that are passed into the component as well as its resulting actions.
+Some other addons are detecting the test framework based on the installed
+addon names and are expecting `ember-cli-qunit` instead. If you have issues
+with this then `ember install ember-cli-qunit`, which should work exactly
+the same.
 
-Component integration tests have the following features:
-- Your test context `this` acts as the outer context for the component.  As a result, you can call `this.set` and `this.on` to setup values and event listeners that you can then have interact with the component.
-- You are required to render the component as a template, e.g. ``this.render(hbs`{{ your-component-name value=value action="updated" }}`)``.  You can render other components as well as block content.
-- All of the normal Ember lifecycle hooks for a component are called (including the new ones from 1.13.x).
-- Testing the component's template is through `this.$()`.
-- You do not require dependencies through `needs:`.  Doing so will force the test into unit mode.
-- You do not have direct access to the component instance.  (`this.subject()` will raise an exception).
 
-### Component Unit Tests
+Usage
+------------------------------------------------------------------------------
 
-[Ember Guide](http://guides.emberjs.com/v1.13.0/testing/testing-components/)
+The following section describes the use of ember-qunit with the latest modern
+Ember testing APIs, as laid out in the RFCs
+[232](https://github.com/emberjs/rfcs/blob/master/text/0232-simplify-qunit-testing-api.md)
+and
+[268](https://github.com/emberjs/rfcs/blob/master/text/0268-acceptance-testing-refactor.md).
 
-```js
-import { test, moduleForComponent } from 'ember-qunit';
+For the older APIs have a look at our [Legacy Guide](docs/legacy.md).
 
-moduleForComponent('x-foo', {
-  unit: true,
-  needs: ['helper:pluralize-string']
-});
+### Setting the Application
 
-// run a test
-test('it renders', function(assert) {
-  assert.expect(1);
+Your `tests/test-helper.js` file should look similar to the following, to
+correctly setup the application required by `@ember/test-helpers`:
 
-  // creates the component instance
-  var subject = this.subject();
+```javascript
+import Application from '../app';
+import config from '../config/environment';
+import { setApplication } from '@ember/test-helpers';
 
-  // render the component on the page
-  this.render();
-  assert.equal(this.$('.foo').text(), 'bar');
-});
+setApplication(Application.create(config.APP));
```

Comment:
I think the `start()` call is missing here. I removed it from the ember-mocha docs, because there is no for mocha. But this here should match the file from the latest blueprints: https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/tests/test-helper.js